### PR TITLE
M3-4137 Cloud Manager-Screen Reader Interoperability Issues

### PR DESCRIPTION
**Pull request by WilkinsKa1**
## Description

A user who utilizes NVDA had trouble accessing the nav and traversing the items. To fix this, I added another skip link and made adjustments to the navigation markup. To do this, I am suggesting also removing the forced focus on h1 elements so whenever page is reloaded, SR users are given the option to immediately focus to navigation or main content rather than being forced into main content and then having to re-tab to navigation again. Curious about what others think of this change. 

I also plan to make the skip links visible on focus for users who are using visual cues in tandem with their SR. 

## Type of Change
- Bug fix ('fix')

If the any above types of change apply to this pull request, please ensure to include one of the listed keywords in the pull request title.

## Todo:
- [x] Audit JAWS
- [ ] Add visible on focus to skip links

## Applicable E2E Tests

To run relevant E2E tests, run these commands in 3 separate terminals:

1. `yarn && yarn start`
2. `yarn selenium`
3. `yarn e2e --spec=COMMA_SEPARATED_LIST_OF_SPECS_HERE --browser=headlessChrome`

## Note to Reviewers

Please test this with NVDA + Firefox and/or JAWS + Edge.

### CODE REVIEW

**Comment by Jskobos** [comment]
Do we have VMWare license keys? My trial has expired.

File: packages/manager/src/App.tsx [hunk 1]
```diff
@@ -160,7 +160,11 @@ export class App extends React.Component<CombinedProps, State> {
     return (
       <React.Fragment>
         {/** Accessibility helpers */}
-        <a href="#main-content" hidden>
+        {/* TODO these should become visible on focus as per: https://www.w3.org/TR/WCAG20-TECHS/G1 */}
+        <a href="#main-navigation" className="visually-hidden">
+          Skip to main navigation
+        </a>
+        <a href="#main-content" className="visually-hidden">
           Skip to main content
```
Comment (Jskobos) [inline packages/manager/src/App.tsx:168]: The navigation link works, but when I hit enter on Skip to main content then start tabbing I end up in the nav bar again. Not sure what the intended behavior is.

Comment (WilkinsKa1) [reply packages/manager/src/App.tsx:168]: You've got it right! Expected behavior is to immediately put focus into the main content region, and should start reading off headings/tabbable content as you go. I wasn't able to reproduce this for both CMR and non-CMR content (using JAWS/Edge), which browser/SR were you using so I can try to replicate?

Comment (Jskobos) [reply packages/manager/src/App.tsx:168]: Edge and Jaws

Comment (Jskobos) [reply packages/manager/src/App.tsx:168]: Occurs to me that I'm using the PR build: does linode-cloud-pr-6527.website-us-east-1.linodeobjects.com have the most recent commits? How do you run stuff locally in Fusion?

**Pull request merged**